trunk/NDHMS/Land_models/NoahMP/IO_code/module_NoahMP_hrldas_driver.F

```diff
@@ -1837,7 +1842,9 @@ subroutine land_driver_exe(itime)
 end subroutine land_driver_exe
 
 !!===============================================================================
-subroutine  ldas_output()
+subroutine  ldas_output(itime)
+integer, intent(in)  :: itime ! time step of the LSM, had to explicitly declare it since 
+                              ! it did not exist in the land_driver_ini 
```

Comment:
👍 The comment is good, but notice how this kind of comment goes stale "had to explicitly..."
I love keywords. I wish they were slightly easier to use in fortran.